### PR TITLE
June 2020 update

### PR DESCRIPTION
**Pull request by darkstarclown**

### CODE REVIEW

File: Hands-on lab/HOL step-by-step - Continuous delivery in Azure DevOps.md [hunk 1]
```diff
@@ -515,32 +516,90 @@ In this exercise, you will create and configure an Azure DevOps account along wi
...
 
-1. Open the *Azure Cloud Shell* to the folder where the Student Files were unzipped (e.g. studentfiles). Then, navigate to the **tailspintoysweb** folder which contains the source code for our web application.
+In this Task, you will configure the Azure DevOps with a Service Connection that allows Azure Dev Ops to securely connect to the resource group you just created in Azure.   
 
-    > **Note**: If this folder doesn't exist ensure you followed the instructions in the 'Before the hands-on lab'.
+**Before continuing, make sure that you are signed in to both the Azure Portal and Azure DevOps using the same Microsoft account**!
 
-    >**Note**: If you are using the Azure Cloud Shell you will be prompted for credentials when using Git. The best way to authenticate is to use a [personal access token](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate), PAT, with a scope Code, Full permissions. Then use that PAT as password (leave user name empty) when prompted.    
+1. In Azure DevOps, ensure you are in the project that you just created, and from the bottom corner of the page, select **Project settings**..
+
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Continuous delivery in Azure DevOps.md:527]: Please review the following alt-texts.  Images are different, alt-text is the same for all of them.  What the alt-text says is highlighted is not.

File: Hands-on lab/HOL step-by-step - Continuous delivery in Azure DevOps.md [hunk 2]
```diff
@@ -515,32 +516,90 @@ In this exercise, you will create and configure an Azure DevOps account along wi
...
+    **Subscription:**	Choose your Azure subscription
+
+    **Resource Group:**	Choose the resource group you created earlier
+    
+    **Service Connection Name:**	 Enter a string value such as "LabConnection"  so you can find this later. 
+
+    Ensure **Grand access permissions to all pipelines** is checked
+
+    >**Note**  During the Service Connection creation process, you might be prompted to sign in to your Microsoft account if Azure DevOps detects it requires authentication.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Continuous delivery in Azure DevOps.md:567]: Missing colon after Note:

File: Hands-on lab/HOL step-by-step - Continuous delivery in Azure DevOps.md (hunk 1 above)
Comment (darkstarclown) [reply Hands-on lab/HOL step-by-step - Continuous delivery in Azure DevOps.md:527]: Hello Dawnmarie.   Just reviewing this.   Paul is OOO for now so I will get latest and fix.  I will need your guidance on the process as Paul has been handling that.   I assume I will fix and run up another PR?   You can reach me at moliver@opsgility.com.   I'm Redmond TZ.

Comment (darkstarclown) [reply Hands-on lab/HOL step-by-step - Continuous delivery in Azure DevOps.md:527]: Some of the alt-texts are essentially the same view of the same application, so could be the same - but we can make them unique.  There are two that stick out to me as incorrect, however.   I can change these.

Comment (DawnmarieDesJardins) [reply Hands-on lab/HOL step-by-step - Continuous delivery in Azure DevOps.md:527]: Hello!  You can actually fix in this PR.  Go to Files Changed tab, find the document, click on the ..., then edit.  That will avoid a 2nd PR and possible merge conflicts.  I know there's another way to push to this PR but I don't use your tools so I'm not sure how to do that, sorry. 

I can look again but, if I remember correctly, the alt-text would say "something" was highlighted...but what was highlighted in the image was different.

Comment (darkstarclown) [reply Hands-on lab/HOL step-by-step - Continuous delivery in Azure DevOps.md:527]: Agree...  that's what I was thinking too on alt-text and I have a fix will drop changes on the file in a bit.   Thanks for the feedback and please reach out if we can provide further help.

Comment (DawnmarieDesJardins) [reply Hands-on lab/HOL step-by-step - Continuous delivery in Azure DevOps.md:527]: Not sure if you understood what I meant sorry.  I see you made some changes but they weren't what I was referring too.  Line 532, the alt-text says, "Repos is highlighted in the left-hand navigation."  In the image, Project settings has the red box around it. Line 536 says the same, only the image is different.  It's an image of project details and Service connections has the red box.  The alt text entered doesn't match the instruction or the image it's connected to.

File: Hands-on lab/HOL step-by-step - Continuous delivery in Azure DevOps.md (hunk 2 above)
Comment (DawnmarieDesJardins) [reply Hands-on lab/HOL step-by-step - Continuous delivery in Azure DevOps.md:567]: I've corrected this.

File: Hands-on lab/HOL step-by-step - Continuous delivery in Azure DevOps.md (hunk 1 above)
Comment (darkstarclown) [reply Hands-on lab/HOL step-by-step - Continuous delivery in Azure DevOps.md:531]: Hi Dawnmarie.   Alt-text fixed and tested.   TY for pointing this out.    LMK if we can assist further.

**Pull request merged**